composer.json

```diff
@@ -1,5 +1,22 @@
 {
     "require-dev": {
-        "wp-coding-standards/wpcs": "^2.3"
+        "wp-coding-standards/wpcs": "^2.3",
+        "wpify/scoper": "^3.1"
+    },
+    "config": {
```

Comment:
Add the platform version restriction here as well just to be safe. So:
```json
    "platform": {
        "php": "7.4"
    }
```
inside the config part